tests/local/test_PacerParseAppellateAttachementPageTest.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
+
+import os
+
+from juriscraper.pacer import AppellateAttachmentPage
+from tests import TESTS_ROOT_EXAMPLES_PACER
+from tests.local.PacerParseTestCase import PacerParseTestCase
+
+
+class PacerParseAppellateAttachmentPageTest(PacerParseTestCase):
+    def setUp(self):
+        self.maxDiff = 200000
+
+    def test_parsing_appellate_attachment_pages(self):
+        path_root = os.path.join(
+            TESTS_ROOT_EXAMPLES_PACER, "appellate_attachment_pages"
```

Comment:
Can you double check that these tests are actually running in CI? We've added stuff like this in the past and later learned some bit of plumbing was missing preventing it from actually running.

Reply:
yep.

Reply:
Confirmed, tests are running in CI.

![Screen Shot 2022-12-09 at 14 18 35](https://user-images.githubusercontent.com/486004/206789300-beddc838-8046-405a-afe1-0d0a99be2bc1.png)